### PR TITLE
Fix Search customer in specific group

### LINKED ISSUE #9825: [BOOM-4203] - BO - Search customer in specific group
> This issue has been migrated from the Forge. Read the [original ticket here](http://forge.prestashop.com/browse/BOOM-4203).

- _**Reporter:**_ salma.moakhar
- _**Created at:**_ Tue, 31 Oct 2017 09:31:07 +0100

<p>Hi,</p>

<p>Navigate to a customer group witch have more than one customer, then try to search with name. On click "Search" you have redirect to list of groups ...not having search results. Going back to the list of customers in the same group, search filter works OK, reset button same work.</p>

<p>Best regards, Salma</p>
        
* How to reproduce the issue ?

N/A

### PR DESCRIPTION
**Pull request by khouloudbelguith**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Navigate to a customer group which has more than one customer, then try to search with a name. On click "Search" you have to redirect to list of groups ...not having search results. Going back to the list of customers in the same group, search filter works OK, reset button same work.
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #9825 
| How to test?  | Navigate to a customer group which has more than one customer, then try to search with a name. On click "Search" => Ok ( Same test with reset)

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/11085)

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
Hello @khouloudbelguith,

you can fix the coding style by executing this command and commit the updated files:

```
./vendor/bin/php-cs-fixer fix
```

Mickaël

**Comment by khouloudbelguith** [comment]
Hi @mickaelandrieu,

Thanks for your review!
Done.

**Review by jolelievre — DISMISSED** [review]
Thank you @khouloudbelguith

File: controllers/admin/AdminGroupsController.php [hunk 1]
```diff
@@ -186,6 +186,10 @@ public function initProcess()
             if (isset($_POST['submitReset' . $this->list_id])) {
                 $this->processResetFilters();
             }
+
+            if (Tools::getIsset('submitFilter')) {
```
Comment (PierreRambaud) [inline controllers/admin/AdminGroupsController.php]: Why not also check for `Tools::getIsset('id_group')`?

Comment (PierreRambaud) [reply controllers/admin/AdminGroupsController.php]: And should it be?
```suggestion
            if (Tools::isSubmit('submitFilter')) {
```

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
See my comment

**Comment by khouloudbelguith** [comment]
Hi @PierreRambaud, @marionf this PR is replaced by this https://github.com/PrestaShop/PrestaShop/pull/14977

Thanks!

**Pull request closed without merging**